dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/PullRequests.java

```diff
@@ -64,9 +65,11 @@ protected boolean isRenovateEnabled(List<String> filePaths, GitHubContentToProce
                 //If the file has the key 'enabled' set to false, it indicates that while the repo has been onboarded to renovate, it has been disabled for some reason
                 return readJsonFromContent(fork.getParent().getFileContent(filePath)).optBoolean("enabled", true);
             } catch (FileNotFoundException e) {
-                log.debug("The file with name {} not found in the repository. Exception: {}", filePath, e.getMessage());
+                log.debug("The file with name {} not found in the repository.Returning false. Exception: {}", filePath, e.getMessage());
```

Comment:
FileNotFoundException error should be in the debug log because we are expecting this for most of the repos. But do we want to do that for IO and JSON exceptions as well? We should log it in warn mode if they are expected to be raised in corner cases

Reply:
@jeetchoudhary Updated